core/contracts/common/implementation/MultiRole.sol

```diff
@@ -10,17 +10,17 @@ library Exclusive {
         return roleMembership.member == memberToCheck;
     }
 
-    function resetMember(RoleMembership storage roleMembership, address newMember) internal {
+    function resetExclusiveMember(RoleMembership storage roleMembership, address newMember) internal {
         require(newMember != address(0x0), "Cannot set an exclusive role to 0x0");
         roleMembership.member = newMember;
     }
 
-    function getMember(RoleMembership storage roleMembership) internal view returns (address) {
+    function getExclusiveMember(RoleMembership storage roleMembership) internal view returns (address) {
```

Comment:
In `scripts/cli/wallet/getTwoKeyContract` and `scripts/TransferPermissions`

Reply:
I think this is the same as above in that the base `MultiRole` function has not changed it's name so should still be fine. Do you agree?